_i18n/en/factors/deployment.md

```diff
@@ -8,7 +8,7 @@ In an ideal world, to release an app update you would
 - automatically install all dependencies (e.g. Xcode and [CocoaPods](https://cocoapods.org))
 - run the deployment script (e.g. [fastlane](https://fastlane.tools))
 
-Unfortunately, due to the fact that Xcode has to run on macOS, and macOS virtualization comes with technical as well as legal challenges, we can't use this approach ([rdar://40669395](https://openradar.appspot.com/radar?id=4929082424819712)). One third party tool that makes it possible to have consistent macOS environments is [Veertu](https://veertu.com/).
+Unfortunately, due to the fact that Xcode has to run on macOS, we can't use this approach ([rdar://40669395](https://openradar.appspot.com/radar?id=4929082424819712)). Running macOS in a virtual environment comes with technical as well as legal challenges. There is a 3rd party commercial solution called [Veertu](https://veertu.com/) that makes it possible to have consistent macOS environments. It still requires Apple hardware to run, but it may be a good option to help you manage your build environments.
```

Comment:
Let's simplify to follow a similar language style to the rest of the page 👍 

```
Unfortunately, due to the fact that Xcode has to run on macOS, we can't use this approach ([rdar://40669395](https://openradar.appspot.com/radar?id=4929082424819712)). Running macOS in a virtual environment comes with technical as well as legal challenges. There is a 3rd party commercial (closed source) solution called [Veertu](https://veertu.com/) that allows you to generate virtaul macOS environments on Apple hardware.
```

Thoughts?

Reply:
Sounds great to me! Thanks @KrauseFx!

Reply:
Do you want to update the PR to use the new language, so that you're listed as the author of this change? :)